x-pack/test/functional/apps/rollupjob/rollup_jobs.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
+import { indexBy } from 'lodash';
+export default function ({ getService, getPageObjects }) {
+  const esArchiver = getService('esArchiver');
+  const testSubjects = getService('testSubjects');
+  const log = getService('log');
+  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'header']);
+
+
+  describe('rollup_jobs', async () => {
+    before(async () => {
+      // init data
+      await Promise.all([
+        esArchiver.loadIfNeeded('logstash_functional'),
+        esArchiver.load('canvas/default'),
+      ]);
+      await PageObjects.common.navigateToApp('rollupjob');
+    });
+
+    after(async () => await esArchiver.unload('logstash_functional'));
+
+    it('create and and save a new job', async () => {
+      await PageObjects.rollup.createNewRollUpJob();
+      await testSubjects.exists('createRollupStep1--active');
+      const jobName = 'Testjob1';
```

Comment:
The test is awesome! I think it is good to move test data (jobName, indexPattern, etc.) to the top for better readability and future updates:
```
const jobName = 'Testjob1';
const indexPattern = '.kibana*';
const indexName = 'rollup_index';
const interval = '1000ms';

Reply:
++ to this suggestion